### ISSUE TITLE
[Request] Regarding importing liked lists to library

### ISSUE THREAD

**Issue by rizreflects** [123Venom/plugin.video.venom#109]
Currently, when you import your trakt liked lists to library, Venom adds the movies but not the list title it originally came with. Would it be possible to also import the name of the list so that it creates a playlist automatically similar to tmdbhelper? 

Right now the only way around this is to create playlists from scratch in Kodi which can be quite time consuming. 

Thanks for all your work.

**Comment by 123Venom** [comment]
Not 100% sure I'm following you.  Import all the liked list items so the library has a named list and the items are in that?  Not sure I've mucked with titles like that in the library, if that is what you mean?  To be honest if TMDb Helper is already doing what you want I would suggest sticking with that as currently I've had little interest in further developing Library Integration.  I've been slowly trying to cut my time investment down and the main area I've focused on is features I do not use.  That is a huge reason why free hosters support was removed, have not used one in prob 5yrs.  I've never once used Library Integration from an addon for daily use, testing for support is another matter..lol.  I'm even considering removing Library Integration from Venom.

**Comment by rizreflects** [comment]
I did try TMDBhelper initially for library integration, but it didn't work very well and would stop updating. Venom on the hand is perfect in this regard.. I've never looked back. That being said, if you don't personally use library integration I'd rather not waste your time. I was under the impression that you used it yourself. Hopefully you'll consider keeping it around though, i'd personally appreciate it a lot. Either way, thanks for all your work.

**Comment by 123Venom** [comment]
No prob man..it's honestly stuck around as long as it did for you all to benefit from and me as a good pastime hobby.  After 3yrs however I've just been trying to cut my free time investment down to what I daily use.  I may keep it around but I'm sorry that I just do not see myself further developing it.  I did use local library content years back for a spell but ONLY for a local media attached eSATA 16tb enclosure, never streaming sources.  I've not ripped a title now in 5yrs so the likely hood I ever will again is doubtful.  Rather sad I got 12tb of 480p movies nobody wants..lol...that's technology for ya.
